src/TetherContent.js

```diff
@@ -28,8 +28,9 @@ class TetherContent extends React.Component {
     super(props);
 
     this.handleDocumentClick = this.handleDocumentClick.bind(this);
-    this.checkTogglerTarget = this.checkTogglerTarget.bind(this);
     this.toggle = this.toggle.bind(this);
+    this.checkTogglerTarget = this.checkTogglerTarget.bind(this);
```

Comment:
Needed to bind these methods, otherwise they'd get called multiple times in the tests.